exercises/e-data02-derive-master.ipynb

```diff
@@ -0,0 +1,866 @@
+{
```

Comment:
<p>Somethings that's not really present throughout this exercise are <em>real-code examples</em>. Add some more code cells that "just work", to give the reader some examples of working code. This is because we don't haven an external reading that illustrates what correct code looks like.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/zdelrosario/evc-course/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>